io.catenax.request_for_quotation/1.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,654 @@
+   samm:dataType :PartObject .
+
+:BillOfMaterialCharacteristic a samm:Characteristic ;
+   samm:preferredName "bill of material characteristic"@en ;
+   samm:description "reference to the single level bill of material as built"@en ;
+   samm:dataType :SingleLevelBomAsBuiltReference .
+
+:DeliveryDateCharacteristic a samm:Characteristic ;
+   samm:preferredName "delivery date characteristic"@en ;
+   samm:description "delivery date"@en ;
+   samm:dataType xsd:date .
+
+:CommentCharacteristic a samm:Characteristic ;
+   samm:preferredName "comment characteristic"@en ;
+   samm:description "placeholder for any kind of additional comments"@en ;
+   samm:dataType xsd:string .
+
+:SetOfFilesCharacteristic a samm:Characteristic ;
+   samm:preferredName "set of files characteristic"@en ;
+   samm:description "in case there are additional documents or files to share"@en ;
+   samm:dataType :File .
+
+:SenderNameCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender name characteristic"@en ;
+   samm:description "name of the sender"@en ;
+   samm:dataType xsd:string .
+
+:SenderAddressCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender address characteristic"@en ;
+   samm:description "address of sender"@en ;
+   samm:dataType xsd:string .
+
+:SenderCompanyNameCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender company name characteristic"@en ;
+   samm:description "company name of sender"@en ;
+   samm:dataType xsd:string .
+
+:SenderEMailCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender eMail characteristic"@en ;
+   samm:description " e-mail address of sender"@en ;
+   samm:dataType xsd:string .
+
+:SenderPhoneNumberCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender phone number characteristic"@en ;
+   samm:description "phone number of sender"@en ;
+   samm:dataType xsd:string .
+
+:SenderAccountAddressCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender account address characteristic"@en ;
+   samm:description "address of sender for account"@en ;
+   samm:dataType xsd:string .
+
+:SenderDeliveryAddressCharacteristic a samm:Characteristic ;
+   samm:preferredName "sender delivery address characteristic"@en ;
+   samm:description "address of sender for delivery"@en ;
+   samm:dataType xsd:string .
+
+:DeliveryRequirementsCharacteristic a samm:Characteristic ;
+   samm:preferredName "delivery requirements characteristic"@en ;
+   samm:description "any kind of delivery requirements"@en ;
+   samm:dataType xsd:string .
+
+:ContextCharacteristic a samm:Characteristic ;
+   samm:preferredName "context characteristic"@en ;
+   samm:description "description of use case context"@en ;
+   samm:dataType xsd:string .
+
+:CatenaXIdTraitCharacteristic a samm-c:Trait ;
+   samm:description "Trait to ensure UUID v4 data format"@en ;
+   samm-c:baseCharacteristic :Uuidv4Characteristic ;
```

Comment:
Here also you can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR45

Reply:
has been added